docs/source/quick_start/install.rst

```diff
@@ -0,0 +1,257 @@
+
+.. caution::
+
+    **DO NOT** download the source files of MCDR and execute them directly, unless you're a developer of MCDR and you know what you are doing
+
+Install using pip
+-----------------
+
+MCDR is available in `PyPI <https://pypi.org/project/mcdreforged>`__, it's reasonably to be installed by:
+
+.. tab:: Windows
+
+    .. code-block:: bat
+
+        pip install mcdreforged
+
+.. tab:: Linux
+
+    .. code-block:: bash
+
+        pip3 install mcdreforged
+
+And upgraded by:
+
+.. tab:: Windows
+
+    .. code-block:: bat
+
+        pip install mcdreforged -U
+
+.. tab:: Linux
+
+    .. code-block:: bash
+
+        pip3 install mcdreforged -U
+
+Verify the installation with:
+
+.. prompt-mcdr-version:: bash $ auto
+
+    $ mcdreforged
+    MCDReforged v@@MCDR_VERSION@@
+
+Externally managed environment
+------------------------------
+
+If you're using Windows, the command above should work fine, MCDR will be installed to global environment - you may ignore this section
+
+For Linux and Mac OS, it's not recommended to install MCDR system-wide (with root), because it can cause conflicts with other Python packages and affect system dependencies
+
+System-wide install also makes version management difficult 
+and requires administrator privileges, increasing security risks
+
+System-wide install may even result in an ``externally-managed-environment`` error
+
+See `PEP 668 <https://peps.python.org/pep-0668/>`__ for the detailed specification
+
+It's safer to keep the installation isolated. As workarounds, there're multiple options for you. For a conclusion:
+
+.. list-table::
+    :header-rows: 1
+
+    * - Method
+      - Pros
+      - Cons
+    * - pip
+      - Native, always available
+      - Not isolated, may affect global packages with root privileges
+    * - pipx
+      - Simplest
+      - 3rd party, different command set
+    * - venv
+      - Native support
+      - Requires manual environment activation
+    * - docker
+      - Reliable across environments
+      - More dependencies and disk space, convoluted learning path
+    * - system package
+      - \-
+      - Same as pip, **not recommended**
+
+Using pipx
+~~~~~~~~~~
+
+Maybe the simplest solution for most users, but requires to use the third-party tool `pipx <https://pipx.pypa.io/>`__, which is designed to installing and running Python applications in isolated environments
+
+Install pipx refer to its `official documentation <https://pipx.pypa.io/stable/#install-pipx>`__
+
+Then install MCDR using pipx:
+
+.. prompt:: bash
+
+    pipx install mcdreforged
+
+When a new version of MCDR available, you may take the upgrade by:
+
+.. prompt:: bash
+
+    pipx upgrade mcdreforged
+
+.. asciinema:: resources/pipx.cast
+    :rows: 8
+
+|
+
+.. note::
+
+    In this way, MCDR will be installed in an isolated environment. Instead of ``pip install <package_name>``, Python packages required by MCDR plugins should be installed by:
```

Comment:
```suggestion
    In this way, MCDR will be installed in an isolated environment. Instead of using ``pip install <package_name>``, Python packages required by MCDR plugins should be installed by:
```